---a/net/minecraft/entity/ai/EntityJumpHelper.java
+++b/net/minecraft/entity/ai/EntityJumpHelper.java
@@ -2,26 +2,23 @@
 
 import net.minecraft.entity.EntityLiving;
 
-public class EntityJumpHelper
-{
+public class EntityJumpHelper {
+
     private EntityLiving entity;
     protected boolean isJumping;
     private static final String __OBFID = "CL_00001571";
 
-    public EntityJumpHelper(EntityLiving p_i1612_1_)
-    {
+    public EntityJumpHelper(EntityLiving p_i1612_1_) {
         this.entity = p_i1612_1_;
     }
 
-    public void setJumping()
-    {
+    public void setJumping() {
         this.isJumping = true;
     }
 
-    public void doJump()
-    {
+    public void doJump() {
         this.entity.setJumping(this.isJumping);
         this.isJumping = false;
     }
-}
+}
